lib/games_teams_collection.rb

```diff
@@ -59,21 +59,23 @@ def percentage_ties
   end
 
   def number_of_wins
-    teams = @games_teams.group_by {|game_team| game_team.team_id}
-    teams.values.map do |game|
-      game.count do |game|
-        game.result == "WIN"
+    winner_goals = []
+    @games_teams.each do |game_team|
+      if game_team.result == "WIN"
+        winner_goals << game_team.goals.to_i
       end
     end
+    winner_goals
   end
 
   def number_of_losses
```

Comment:
Could we rename this so it reflects what it's doing now?

Reply:
it is finding the number of losses